tests/prettier/jsx_long_name_closed.js

```diff
@@ -0,0 +1 @@
+<BaseForm url="/auth/google" method="GET" colour="blue" size="large" submitLabel="Sign in with Google"/>
```

Comment:
The tests situation is a little weird, but I'm hesitant to touch much outside of the `prettier` directory because all of those tests are Flow's test suite. In the future we should be able to remove our version and copy in the latest Flow tests.